### PR TITLE
Add table for owners and species

### PR DESCRIPTION
**Pull request by farouqdaif1**
* Create a table named owners with the following columns:
    * id: integer (set it as autoincremented PRIMARY KEY)
    * full_name: string
    * age: integer
* Create a table named species with the following columns:
  * id: integer (set it as autoincremented PRIMARY KEY)
  * name: string
* Modify animals table:
  * Make sure that id is set as autoincremented PRIMARY KEY
  * Remove column species
  * Add column species_id which is a foreign key referencing species table
  * Add column owner_id which is a foreign key referencing the owners table
* Remember all this goes in schema.sql.
* Insert the following data into the owners table:
  * Sam Smith 34 years old.
  * Jennifer Orwell 19 years old.
  * Bob 45 years old.
  * Melody Pond 77 years old.
  * Dean Winchester 14 years old.
  * Jodie Whittaker 38 years old.
* Insert the following data into the species table:
  * Pokemon
  * Digimon
* Modify your inserted animals so it includes the species_id value:
  * If the name ends in "mon" it will be Digimon
  * All other animals are Pokemon
* Modify your inserted animals to include owner information (owner_id):
  * Sam Smith owns Agumon.
  * Jennifer Orwell owns Gabumon and Pikachu.
  * Bob owns Devimon and Plantmon.
  * Melody Pond owns Charmander, Squirtle, and Blossom.
  * Dean Winchester owns Angemon and Boarmon.
* Remember these insertions and modifications happen in data.sql.
* Write queries (using JOIN) to answer the following questions:
  * What animals belong to Melody Pond?
  * List of all animals that are pokemon (their type is Pokemon).
  * List all owners and their animals, remember to include those that don't own any animal.
  * How many animals are there per species?
  * List all Digimon owned by Jennifer Orwell.
  * List all animals owned by Dean Winchester that haven't tried to escape.
  * Who owns the most animals?

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -53,3 +53,58 @@ VALUES ('Boarmon','2005-06-07',7,true,20.4);
...
+update animals set species_id=2 where name like '%mon';
+--UPDATE 5
+--All other animals are Pokemon
+update animals set species_id=1 where name not like '%mon';
+--UPDATE 4
+--Modify your inserted animals to include owner information (owner_id):
+--Sam Smith owns Agumon.
+update animals set owner_id=1 where name ='Agumon';
+--Jennifer Orwell owns Gabumon and Pikachu.
+UPDATE animals SET owner_id=3 WHERE name IN ('Gabumon','Pikachu');
```
Comment (ZeenatLawal) [inline data.sql:104]: - [ ] The owner_id here should be set to 2 as `Jennifer Orwell` is the second entry in the owner's table.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @farouqdaif1, 

Good job so far!:clap: :clap:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎊 🎊 
- Gitflow is followed. ✔️
- PR has a title and summary. ✔️


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 2]
```diff
@@ -230,11 +230,109 @@ SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
...
+ Digimon |     5
+(2 rows)
+*/
+--List all Digimon owned by Jennifer Orwell.
+SELECT animals.name 
+FROM animals 
+JOIN owners 
+ON animals.owner_id=owners.id 
+WHERE owners.full_name='Jennifer Orwell'
+AND species_id=2;
```
[the remark this answers is not in the document]
Comment (farouqdaif1) [reply queries.sql]: thanks

**Review by ZeenatLawal — APPROVED** [review]
Hi @farouqdaif1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**